apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/app-tab.tsx

```
@@ -0,0 +1,238 @@
+      if (configType && configType in noCodeConfigTypeParam) {
+        configTypeLabel = noCodeConfigTypeParam[configType];
+      } else if (configType) {
+        configTypeLabel = configType;
+      }
+
+      return `(${t("environments.surveys.summary.in_app.display_criteria.no_code_trigger")}, ${configTypeLabel})`;
+    }
+  };
+
+  const getSegmentTitle = (segment: TSegment | null) => {
+    if (segment && segment.filters && segment.filters.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)